server/models/clap.js

```diff
@@ -0,0 +1,21 @@
+module.exports = (sequelize, DataTypes) => {
+  const Clap = sequelize.define('Clap', {
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      defaultValue: DataTypes.UUIDV4,
+      unique: true,
+      allowNull: false
+    },
+    articleId: {
+      type: DataTypes.UUID,
+      allowNull: false,
+    },
+    userId: {
+      type: DataTypes.UUID,
+      allowNull: false,
+    }
+  }, {});
+  Clap.associate = (models) => {};
```

Comment:
Good job, but there should be an association between claps and articles created, what do you think?

Reply:
@davidokonji very true. To add to this, I think the route for this should be `api/v1/articles/claps` following RESTful API convention.

Reply:
@davidokonji, the relationships between the claps and articles is out of scope for this pull request. The relationships should be for the person who will work on getting articles

Reply:
Thank you for the feedback @davidokonji, your point is valid, but I realised it is outside the scope of what I am working on.  That can be included for fetching all claps

Reply:
@cupidcoder, I used this as a POST request, I feel the article id should be sent with the request body.

Reply:
@geekdanitek how does creating a relationship between claps and articles have anything to do with fetching articles? Please do the needful.

Reply:
@samorahno The recommendation made by @cupidcoder makes sense. `api/v1/articles/:articleId/claps` is a good representation. I do not want you to put the id in the body. The convention is that the id should be in the url as a parameter

Reply:
Noted @paradisekelechi, this will be implemented right away